innersource-license.md

```diff
@@ -0,0 +1,83 @@
+## Title
+
+InnerSource License
+
+## Patlet
+
+Two companies that belong to the same group want to share software source code with each other but they are concerned about the implications in terms of legal liabilities or cross-company accounting. An **InnerSource License** provides a reusable legal framework for the inter-company agreement within the group, which opens up new collaboration options.
+
+## Problem
+
+When two companies within a group want to share code with each other, they need an agreement about the terms and often a legal contract. Creating such agreements on a per project basis takes effort and creates a barrier for sharing. i.e. a team/company might decide not to share their source code with another company in the group because it seems complicated.
+
+Barriers for sharing can lead to silos and duplication of effort in rebuilding similar solutions in multiple parts of the group.
```

Comment:
```suggestion
This uncertainty creates an effective barrier to InnerSource collaboration among the different legal entities of the organization.
```

I feel the fact that the patterns in this repo are "InnerSource Patterns" establishes the context that we want to bring down silos, avoid duplication, forster reuse and collaboration. I'd suggest we not list them again to avoid duplication.

Reply:
Just like our other thread about the **InnerSource Context**, I would like to keep this in here to allow the readers to understand the pattern without having to read anything else.

@maxcapraro please close this conversation if you are fine with leaving this section as is.